operator/cmd/root.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/microsoft/retina/operator/cmd/legacy"
+	"github.com/spf13/cobra"
+)
+
+const (
+	configFileName = "retina/operator-config.yaml"
+)
+
+var (
+	metricsAddr          string
+	probeAddr            string
+	enableLeaderElection bool
+	cfgFile              string
+
+	rootCmd = &cobra.Command{
+		Use:   "retina-operator",
+		Short: "Retina Operator",
+		Long:  "Start Retina Operator",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println("Starting Retina Operator")
+			d := legacy.NewOperator(metricsAddr, probeAddr, cfgFile, enableLeaderElection)
+			d.Start()
+		},
+	}
+)
+
+func init() {
+	rootCmd.Flags().StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
```

Comment:
Is this required? @huntergregory

Reply:
I believe it's used in the legacy operator. I'd say to keep it just in case it's needed later even if this variable isn't referenced?

Reply:
Actually, if `metricsAddr` and `probeAddr` are already fields in the operator's configmap (since they're used in the legacy operator?) let's remove these pieces from the cobra flags

Reply:
keeping this in this PR.